src/axom/inlet/docs/sphinx/simple_types.rst

```diff
@@ -0,0 +1,139 @@
+############
+Simple Types
+############
+
+To help structure your input file, Inlet categories the information into
+two types: Fields and Tables.
+
+Fields refer to the individual scalar values that are either at the global level or
+that is contained inside of a Table.
+
+Tables can contain multiple Fields, as well as, a single array, and a single dictionary.
+
+.. note::  There is an inherent global Table that holds all global level Fields.  This can be
+  accessed via your `Inlet` class instance.
+
+
+*******
+Fields
+*******
+
+In Inlet, Fields represent an individual scalar value. There are four supported
```

Comment:
I'm not sure if we should use the word "primitive" here (or where I commented above).  It's used all over in the implementation, but it might also be useful to say that Field <-> a primitive.